src/container/srv_target.c

```diff
@@ -772,9 +774,21 @@ cont_child_started(struct ds_cont_child *cont_child)
 	return !d_list_empty(&cont_child->sc_link);
 }
 
+static int cont_close_hdl(uuid_t cont_hdl_uuid);
+
 static void
 cont_child_stop(struct ds_cont_child *cont_child)
 {
+	struct ds_cont_hdl	*hdl;
+
+	while ((hdl = d_list_pop_entry(&cont_child->sc_open_hdls,
+				       struct ds_cont_hdl, sch_link)) != NULL) {
+		D_DEBUG(DF_DSMS, "Force closing container open handle "DF_UUID"/"DF_UUID"\n",
+			DP_UUID(cont_child->sc_uuid), DP_UUID(hdl->sch_uuid));
+
+		cont_close_hdl(hdl->sch_uuid);
+	}
```

Comment:
a bit duplicate with sc_open? or probably replace sc_open with !d_list_empty(&cont_child->sc_open_hdls)?

Reply:
Yes, I will consider to cleanup related logic in subsequent patch.